Automattic/wp-calypso
# Packages: Move Custom CSS Properties to new `calypso-css-custom-properties` package

Preparatory step to move Calypso's build system into an npm.
See https://github.com/Automattic/wp-calypso/pull/30772#issuecomment-465968559 (and conversation below) for background.

#### Questions/Notes
- Do we like the name (`calypso-css-custom-properties`)? Yay? Nay?
- I was originally going to move the entire `assets/stylesheets/shared/` folder (and was going to call the package `calypso-shared-style`), but decided to keep the package as minimal as possible instead. Thoughts?
- [x] ~[`bin/build-packages`](https://github.com/Automattic/wp-calypso/blob/7eea04a17c7a6743d95f7c039e96df7425174a67/bin/build-packages.js) doesn't work for styling-only packages. I think it's too rigid (might also be for different kinds of packages) -- my suggestion would be to make the build process each package's individual concern for greater flexibility, i.e. have `bin/build-packages` just run each package's `npm run build`. IIRC, lerna had a way to do that (but maybe only as part of `bootstrap`).~ Resolved, see #31112 

#### Changes proposed in this Pull Request

* Move Custom CSS Properties to new `calypso-css-custom-properties` package

#### Testing instructions

- `npm run distclean`
- `npm start`

Verify that Calypso runs as before, and that styling isn't broken.
